node/core/dispute-coordinator/src/real/ordering/tests.rs

```diff
@@ -142,24 +145,39 @@ fn get_block_number_hash(n: BlockNumber) -> Hash {
 	BlakeTwo256::hash(&n.encode())
 }
 
+fn make_candidate_receipt(_relay_parent: Hash) -> CandidateReceipt {
+	let zeros = dummy_hash();
+	let descriptor = CandidateDescriptor {
+		para_id: 0.into(),
+		relay_parent: get_block_number_hash(2),
```

Comment:
Why do we pass in a relay parent, but then make one up?

Reply:
Uh, that's a very good point... I was just very happy that this test finally passed again and focused on making the rest work, good catch :+1: